Examples/Cocoa/Sources/Objective_C/Board.m

```diff
@@ -100,6 +102,15 @@ - (instancetype)initWithModelDictionary:(NS_VALID_UNTIL_END_OF_SCOPE NSDictionar
             }
         }
         {
+            __unsafe_unretained id value = modelDictionary[@"contributors"]; // Collection will retain.
```

Comment:
Doesn't this need to be converted to `NSSet`?

Reply:
Good catch. I just added it.